client/app/dedicatedCloud/datacenter/drp/dedicatedCloud-datacenter-drp.constants.js

```diff
@@ -12,23 +13,30 @@ export const DEDICATEDCLOUD_DATACENTER_DRP_UNAVAILABLE_IP_STATUS = [
   'broadcast', 'gateway', 'hsrp',
 ];
 
+/* eslint-disable no-useless-escape */
+export const DEDICATEDCLOUD_DATACENTER_DRP_IP_BLOCK_REG_EXP = new RegExp('^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\/2[2-8]$');
+
 export const DEDICATEDCLOUD_DATACENTER_DRP_IP_USAGE_MAC_ADDRESS_REG_EXP = new RegExp('/^([0-9A-F]{2}[:-]){5}([0-9A-F]{2})$/');
 
 export const DEDICATEDCLOUD_DATACENTER_DRP_STATUS = {
   delivered: 'delivered',
   delivering: 'delivering',
+  disabled: 'disabled',
   disabling: 'disabling',
+  error: 'error',
   provisionning: 'provisionning',
   toDisable: 'toDisable',
   toDo: 'todo',
   toProvision: 'toProvision',
   toUnprovision: 'toUnprovision',
   unprovisionning: 'unprovisionning',
+  waitingActivation: 'waitingActivation',
```

Comment:
Could it make sense to get the Enum status from API and `map` it to this ? That way new statuses would be added dynamically to the list
(Can be done in a another PR if that's the case)

Reply:
This constant isn't used, but you're right that may be obtained from the API directly

Reply:
We should not get Enums from the API at runtime, it should be generated but absolutely not retrieved from the API for each user

Think about users in third world countries like the US who don't have a lot of mobile data